sklearn/metrics/_scorer.py

```diff
@@ -127,42 +132,15 @@ def __init__(self, score_func, sign, kwargs):
         self._score_func = score_func
         self._sign = sign
 
-    @staticmethod
-    def _check_pos_label(pos_label, classes):
-        if pos_label not in list(classes):
-            raise ValueError(
-                f"pos_label={pos_label} is not a valid label: {classes}"
-            )
-
-    def _select_proba_binary(self, y_pred, classes):
```

Comment:
This function was moved within `_get_response` when selecting the binary probability.